java/feathub-connectors/flink-connectors/flink-connector-prometheus/src/main/java/com/alibaba/feathub/flink/connectors/prometheus/PrometheusSinkFunction.java

```diff
@@ -331,6 +359,23 @@ private static String[] split(String str, String delim) {
                 .toArray(String[]::new);
     }
 
+    private void retryUntilTimeout(
+            RunnableWithException runnable, Class<?> occasionalExceptionClass) throws Exception {
```

Comment:
It is more readable to use `retriableException` instead of `occasionalExceptionClass`.

Also, it seems simpler to just try/catch all Exception without passing this class as parameter.